.github/workflows/README.md

```diff
@@ -0,0 +1,66 @@
+| Test full                   | .github/workflows/ci_test-full.yml  | Run all tests except for accelerator-specific, standalone and slow tests.                                                                                                  | CPU           | (3.7, 1.8), (3.7, 1.11), (3.9, 1.8), (3.9, 1.11) | linux, mac, windows |
+| Test with Conda             | .github/workflows/ci_test-conda.yml | Same as ci_test-full.yml but with dependencies installed with conda.                                                                                                       | CPU           | (3.8, 1.8), (3.8, 1.9), (3.8, 1.10), (3.9, 1.11) | linux               |
+| Test slow                   | .github/workflows/ci_test-slow.yml  | Run only slow tests. Slow tests usually need to spawn threads and cannot be speed up or simplified.                                                                        | CPU           | (3.7, 1.8)                                       | linux, mac, windows |
+| PL.pytorch-lightning (IPUs) | .azure-pipelines/ipu-tests.yml      | Run only IPU-specific tests.                                                                                                                                               | IPU           | (3.8, 1.9)                                       | linux               |
+| PL.pytorch-lightning (HPUs) | .azure-pipelines/hpu-tests.yml      | Run only HPU-specific tests.                                                                                                                                               | HPU           | (3.8, 1.10)                                      | linux               |
+| PL.pytorch-lightning (GPUs) | .azure-pipelines/gpu-tests.yml      | Run all CPU and GPU-specific tests, standalone and examples. Each standalone test needs to be run in separate processes to avoid unwanted interactions between test cases. | GPU           | (3.7, 1.8)                                       | linux               |
+| PyTorchLightning.Benchmark  | .azure-pipelines/gpu-benchmark.yml  | Run speed/memory benchmarks for parity with pure PyTorch.                                                                                                                  | GPU           | (3.7, 1.8)                                       | linux               |
+| test-on-tpus                | .circleci/config.yml                | Run only TPU-specific tests.                                                                                                                                               | TPU           | (3.7, 1.9)                                       | linux               |
+
+- \*Accelerators used in CI
+  - GPU: 2 x NVIDIA P100
+  - TPU: Google GKE TPUv2/3
```

Comment:
```suggestion
  - TPU: Google GKE TPUv2
```
@kaushikb11 atm we use only v2, correct?

Reply:
Nope, it's TPU v3 https://github.com/PyTorchLightning/pytorch-lightning/blob/master/dockers/tpu-tests/tpu_test_cases.jsonnet#L19